tests/acceptance/routing-test.js

```diff
@@ -5,58 +5,136 @@ import startApp from 'ember-twiddle/tests/helpers/start-app';
 module('Acceptance | routing', {
   beforeEach: function() {
     this.application = startApp();
+
+    this.registerWaiter = () => {
+      window.messagesWaiting = 1;
+      this._waiter = () => {
+        return window.messagesWaiting === 0;
+      };
+      Ember.Test.registerWaiter(this._waiter);
+    };
   },
 
   afterEach: function() {
+    window.messagesWaiting = 0;
+    if (this._waiter) {
```

Comment:
The problem with that is that one app sends the communication and a different app receives it.